autopush/utils.py

```diff
@@ -54,6 +58,65 @@ def generate_hash(key, payload):
     return h.hexdigest()
 
 
+def parse_header(header, major=";", minor="="):
+    """ Convert a multi-component header line (e.g. "a;b=c;d=e;...") to
+    a list.
+
+    For example, if the header content line is
+
+    `"a;c=1;b;d=2+3=5"`
```

Comment:
If cyclone doesn't strip quoting, it's also possible this could be called with:
```
'a;c="fred";d="smith"'
```
It should strip quotes from the value.